### PR TITLE
Added explicit commentary to .card-* classes

### LINKED ISSUE #17398: Many .card-* classes lack prose docs
Currently, many of them are just used in the docs examples, without accompanying explicit commentary. Each of them should at least be part of some sentence that explains what they are and where they're permitted. 
CC: Cardcaptor @mdo

### PR DESCRIPTION
**Pull request by florisverburg**
This should resolve #17398

### CODE REVIEW

File: docs/components/card.md [hunk 1]
```diff
@@ -79,6 +85,10 @@ Cards support a wide variety of content, including images, text, list groups, li
 </div>
 {% endexample %}
 
+Subtitles are used by adding a `.card-subtitle` to a `<h*>` tag. If the `.card-title` and the `.card-subtitle` items are placed in a `.card-block` item, the card title and subtitle are aligned nicely.
```
Comment (XhmikosR) [inline docs/components/card.md]: Shouldn't this be **an** `<h*>` tag?

File: docs/components/card.md [hunk 2]
```diff
@@ -79,6 +85,10 @@ Cards support a wide variety of content, including images, text, list groups, li
...
+
+The multiple content types can easily be combined to create the card you need. See below for an example.
```
Comment (XhmikosR) [inline docs/components/card.md]: Isn't "can be easily combined" better?

**Comment by florisverburg** [comment]
@XhmikosR, you're completely right! I've fixed these typo's.

**Comment by florisverburg** [comment]
How about the rest? Is it ready to be merged or should I change anything else?

**Pull request merged**